infrastructure/docker/services/php/base/php-configuration/mods-available/sudo.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+export GOSU_PLEASE_LET_ME_BE_COMPLETELY_INSECURE_I_GET_TO_KEEP_ALL_THE_PIECES="I've seen things you people wouldn't believe. Attack ships on fire off the shoulder of Orion. I watched C-beams glitter in the dark near the Tannhäuser Gate. All those moments will be lost in time, like tears in rain. Time to die."
+exec gosu 0:0 $@
```

Comment:
Not sure to get why this script is in mods-available dir?

Reply:
Typo :) Thanks